packages/flutter_tools/lib/src/usage.dart

```diff
@@ -75,9 +75,15 @@ class Usage {
   /// reports coming from the same computer.
   String get clientId => _analytics.clientId;
 
-  void sendCommand(String command) {
-    if (!suppressAnalytics)
-      _analytics.sendScreenView(command);
+  void sendCommand(String command, { Map<String, String> parameters }) {
+    if (suppressAnalytics)
+      return;
+
+    if (parameters != null) {
+      parameters.forEach(_analytics.setSessionValue);
```

Comment:
This will add these params to the analytics session - every call after this will also report these params (`cd3`, `cd4`, ...).

I think what we want is to have the ability to specify arbitrary additional fields per analytics call, or to expose the `callRaw` method in package:usage.